src/Components/Publishing/Sections/Caption.tsx

```diff
@@ -57,6 +57,7 @@ const Figcaption = styled.div<FigcaptionProps>`
     color: ${props =>
       props.color ? props.color : props.layout === "classic" ? "#666" : "#999"};
     margin: 0;
+    display: inline;    
```

Comment:
Would recommend double checking that this renders as expected in Positron once merged.